src/main/java/uk/gov/pay/products/client/publicapi/model/Link.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.products.client.publicapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(value={ "params" })
+public class Link {
+
+    private String href;
+    private String method;
+    private String type;
```

Comment:
Here I would be inclined to leave it as a string for convenience. I don't want to list all available types and since we are actually not even using this value it may not provide much value, not immediately anyway.